Cargo.toml

```diff
@@ -38,12 +38,13 @@ targets = [
     "i686-pc-windows-msvc",
     "x86_64-pc-windows-msvc",
     # macOS
+    "aarch64-apple-darwin",
     "x86_64-apple-darwin",
     # Unix (X11 & Wayland)
     "i686-unknown-linux-gnu",
     "x86_64-unknown-linux-gnu",
     # iOS
-    "x86_64-apple-ios",
+    "aarch64-apple-ios",
```

Comment:
I saw that e.g. [`mio`](https://docs.rs/mio/1.0.2/mio/index.html) is able to successfully deploy `aarch64-apple-ios` on Docs.rs, I assume this should be fine?

Reply:
Yeah, I think it is, docs.rs is cross-compiling these anyhow